include/yaml-cpp/node/convert.h

```diff
@@ -241,6 +242,35 @@ struct convert<std::list<T> > {
   }
 };
 
+// std::array
+template <typename T, std::size_t S>
```

Comment:
I'd use N for the length, rather than S, so it looks more like a number rather than a type.